pkg/stdlib/rows_test.go

```diff
@@ -35,6 +35,11 @@ func TestRows(t *testing.T) {
 			Columns: []string{"c1"},
 			Values:  []*schema.SQLValue{{Value: nil}},
 		}},
+		columns: []*schema.Column{
+			{
+				Name: "c1",
+			},
+		},
 	}
 
 	ast := r.Columns()
```

Comment:
it would be good having a test case with empty rows and still returning the list of columns

Reply:
ack